### PR TITLE
 deploy

### PR DESCRIPTION
**Pull request by ToluClassics**
Hi @philschmid , Thank you for this! really useful

Ran into some minor errors when trying to reproduce. 
Just added a little fix in case somebody else is trying to reproduce

### CODE REVIEW

File: code/inference.py [hunk 1]
```diff
@@ -14,6 +14,8 @@ def model_fn(model_dir):
     else:
         device = -1
 
-    generation = pipeline("text-generation", model=model, tokenizer=tokenizer, device=device)
+    generation = pipeline(
+        "text-generation", model=model, tokenizer=tokenizer, device=device
+    )
 
-    return generation
+    return
```
Comment (philschmid) [inline code/inference.py]: why did you remove `generation` ?  you need to return the model in the `model_fn`

Comment (ToluClassics) [reply code/inference.py]: Yeah, I must have deleted mistakenly, Updated it now.

Comment (philschmid) [reply code/inference.py]: thanks!

**Review by philschmid — APPROVED** [review]
Thank you for fixing those little bugs
